### PR TITLE
Eventhubs README

### PR DESCRIPTION
**Pull request by YijunXieMS**
README updates of azure-eventhub after code change

### CODE REVIEW

File: sdk/eventhub/azure-eventhubs/README.md [hunk 1]
```diff
@@ -116,7 +117,7 @@ The following sections provide several code snippets covering some of the most c
 - [Consume events from an Event Hub](#consume-events-from-an-event-hub)
 - [Async publish events to an Event Hub](#async-publish-events-to-an-event-hub)
 - [Async consume events from an Event Hub](#async-consume-events-from-an-event-hub)
-- [Consume events using a partition manager](#consume-events-using-a-partition-manager)
+- [Consume events and save checkpoint using a checkpoint store](#consume-events-and-save-checkpoint-using-a-checkpoint-store)
```
Comment (annatisch) [inline sdk/eventhub/azure-eventhubs/README.md]: "save checkpoints"

File: sdk/eventhub/azure-eventhubs/README.md [hunk 2]
```diff
@@ -200,59 +182,32 @@ with client:
 
 ### Async publish events to an Event Hub
```
Comment (annatisch) [inline sdk/eventhub/azure-eventhubs/README.md]: "Publish events to an Event Hub asynchronously"

File: sdk/eventhub/azure-eventhubs/README.md [hunk 3]
```diff
@@ -269,69 +224,62 @@ import asyncio
...
+### Consume events and save checkpoint using a checkpoint store
 
 `EventHubConsumerClient` is a high level construct which allows you to receive events from multiple partitions at once 
 and load balance with other consumers using the same Event Hub and consumer group.
 
 This also allows the user to track progress when events are processed using checkpoints.
 
 A checkpoint is meant to represent the last successfully processed event by the user from a particular partition of
-a consumer group in an Event Hub instance.The `EventHubConsumerClient` uses an instance of PartitionManager to update checkpoints
+a consumer group in an Event Hub instance.The `EventHubConsumerClient` uses an instance of `CheckpointStore` to update checkpoints
```
Comment (annatisch) [inline sdk/eventhub/azure-eventhubs/README.md]: Missing space: "instance.The"

File: sdk/eventhub/azure-eventhubs/README.md [hunk 4]
```diff
@@ -269,69 +224,62 @@ import asyncio
...
 
-async def do_operation(event):
-    # do some sync or async operations. If the operation is i/o intensive, async will have better performance
-    print(event)
-
-
 async def process_event(partition_context, event):
-    await partition_context.update_checkpoint(event)
+    pass # do something
+    await partition_context.update_checkpoint(event)  # Or update_checkpoint every N events for better performance.
```
Comment (annatisch) [inline sdk/eventhub/azure-eventhubs/README.md]: Remove `pass` here - just have the comment `# do something`.

File: sdk/eventhub/azure-eventhubs/README.md (hunk 1 above)
Comment (YijunXieMS) [reply sdk/eventhub/azure-eventhubs/README.md]: Updated

File: sdk/eventhub/azure-eventhubs/README.md (hunk 2 above)
Comment (YijunXieMS) [reply sdk/eventhub/azure-eventhubs/README.md]: Updated

File: sdk/eventhub/azure-eventhubs/README.md (hunk 3 above)
Comment (YijunXieMS) [reply sdk/eventhub/azure-eventhubs/README.md]: Updated

File: sdk/eventhub/azure-eventhubs/README.md (hunk 4 above)
Comment (YijunXieMS) [reply sdk/eventhub/azure-eventhubs/README.md]: Updated

**Pull request merged**